### PR TITLE
Simplified version of the zipkin receiver with IP detecting ability

### PR DESCRIPTION
**Pull request by owais**
Another PR (#45) was merged earlier to add IP detection support to the Zipkin receiver as it was required by the kubernetes auto-tagging feature. This PR simplifies that code a bit.

Depends on https://github.com/open-telemetry/opentelemetry-collector/pull/473

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -56,3 +56,5 @@ replace contrib.go.opencensus.io/exporter/ocagent => github.com/omnition/opencen
 replace github.com/census-instrumentation/opencensus-proto => github.com/omnition/opencensus-proto v0.2.1-gogo-unary
 
 replace git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
+
+replace github.com/open-telemetry/opentelemetry-collector => github.com/owais/opentelemetry-collector v0.1.1-0.20191221015705-9b7fe3c20df9
```
Comment (dmitryax) [inline go.mod]: Why this dependency change needed?

Comment (owais) [reply go.mod]: It is depending on the PR on upstream Otel collector. This will be replaced once the PR is merged.

**Comment by tigrannajaryan** [comment]
@owais  Can you improve PR description? I do not quite understand the change.

**Comment by owais** [comment]
Updated.

File: go.mod (hunk 1 above)
Comment (tigrannajaryan) [inline go.mod]: Let's wait for dependent PR to be merged so that we don't need to use this.
Otherwise, LGTM.

Comment (owais) [reply go.mod]: Sounds good. That was the plan.

**Pull request closed without merging**